### PR TITLE
FoCI Test App Changes

### PR DESCRIPTION
**Pull request by RPangrle**
* Add application profile switching support to the test app
* Include a description of the authentication result on the acquireToken screen
* Include the family ID (if it exists) of MRRTs on the cache screen
* Include error information on the log statement when a refresh token operation fails

### CODE REVIEW

**Comment by RPangrle** [comment]
@jasoncoolmax

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+//  ADTestAppProfileViewController.h
+//  MyTestiOSApp
+//
+//  Created by Ryan Pangrle on 4/29/16.
+//  Copyright © 2016 MS. All rights reserved.
+//
```
Comment (kpanwar) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.h]: Put correct headers. Can  you check if we missed any other files too in old CRs?

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m [hunk 2]
```diff
@@ -0,0 +1,134 @@
+//
+//  ADTestAppProfileViewController.m
+//  MyTestiOSApp
+//
+//  Created by Ryan Pangrle on 4/29/16.
+//  Copyright © 2016 MS. All rights reserved.
+//
```
Comment (kpanwar) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m]: Put correct headers.

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                           @"resource" : @"https://graph.windows.net",
+                           // NOTE: The settings below should come from your registered application on
+                           //       the azure management portal.
+                           @"clientId" : @"b92e0ba5-f86e-4411-8e18-6b5f928d968a",
+                           @"redirectUri" : @"x-msauth-adaltestapp-210://com.microsoft.adal.2.1.0.TestApp",
+                           },
+       @"Office"      : @{ @"authority" : @"https://login.microsoftonline.com/common",
+                           @"resource" : @"https://api.office.com/discovery",
+                           @"clientId" : @"d3590ed6-52b3-4102-aeff-aad2292ab01c",
+                           @"redirectUri" : @"urn:ietf:wg:oauth:2.0:oob",
```
Comment (kpanwar) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m]: are we not testing broker flow for Office? may be point to word redirect-uri?

Comment (RPangrle) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m]: You can't without building a completely different app, as it's tied to the bundle ID. I include the OneDrive one here so I can manually test out two cases where the broker redirect URI check should fail.

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.h (hunk 1 above)
Comment (RPangrle) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.h]: 👍

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m (hunk 2 above)
Comment (RPangrle) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m]: 👍

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m (hunk 3 above)
Comment (RPangrle) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppProfileViewController.m]: And no, I'm not particularly interested in testing out every possible permutation of input on the broker from the test app. Testing that we can talk to broker and get a response back is sufficient from a local ADAL perspective.

**Comment by kpanwar** [comment]
:shipit:

File: Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m [hunk 4]
```diff
@@ -208,9 +228,12 @@ - (void)acquireTokenInteractive:(ADPromptBehavior)promptBehavior
             return;
         }
         fBlockHit = YES;
-        NSLog(@"result: %@", result);
+        
+
```
Comment (jasoncoolmax) [inline Samples/MyTestiOSApp/MyTestiOSApp/ADTestAppAcquireTokenViewController.m]: shall we avoid multiple blank lines here? :P

**Pull request merged**